src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Taxon.xml

```diff
@@ -45,6 +45,8 @@
         <attribute name="enabled">
             <group>admin:taxon:read</group>
             <group>admin:taxon:create</group>
+        <attribute name="position">
+            <group>admin:taxon:read</group>
             <group>admin:taxon:update</group>
         </attribute>
```

Comment:
```suggestion
        <attribute name="enabled">
            <group>admin:taxon:read</group>
            <group>admin:taxon:create</group>
            <group>admin:taxon:update</group>
        <attribute name="position">
            <group>admin:taxon:read</group>
            <group>admin:taxon:update</group>
        </attribute>
```
I don't see why the ability to turn a taxon off was removed from the update

Reply:
:sweat_smile: I fixed it